### DISCUSSION TITLE
blank tiles on the map (mapsforge 0.18.0)

### DISCUSSION THREAD

**Discussion by ZhukovAlV** [mapsforge/mapsforge#1340] [Q&A] [answered]
A have if rendering lot of blank tiles randomly appear:
![2022-07-22 16-07-57](https://user-images.githubusercontent.com/52224813/180417227-461a2ad3-9e85-49bb-9062-41bc8565bb11.png)

This problem is only observed when the screen is large (full screen). 
And if I 
mapView.getModel().displayModel.setFixedTileSize(256);
change to:
mapView.getModel().displayModel.setFixedTileSize(512);
The problem does not appear. But the tiles are displayed too big.

I read about this issue here https://github.com/mapsforge/mapsforge/issues/806 and https://github.com/mapsforge/mapsforge/issues/868
The following was suggested here:
ReadBuffer.setMaximumBufferSize(6500000);  

But class org.mapsforge.map.reader.ReadBuffer don't have setMaximumBufferSize().

They also wrote to use a single thread. It also didn't solve the problem.

**Comment by devemux86** [comment, marked as answer]
See the desktop example [Samples](https://github.com/mapsforge/mapsforge/blob/master/mapsforge-samples-awt/src/main/java/org/mapsforge/samples/awt/Samples.java).
(it appears that your tile cache is not created correctly)

Also see https://github.com/mapsforge/mapsforge/discussions/1331.

**Reply by ZhukovAlV** [reply]
Thank you very much.
Parameters.SQUARE_FRAME_BUFFER = false;
It helped.
